### PR TITLE
ci: fix release step by separating release to AWS bucket.

### PR DESCRIPTION
**Pull request by Matovidlo**
Jira: [PSGO-668](https://keboola.atlassian.net/browse/PSGO-668)

**Changes:**
- The new cache located in AWS has changed the credentials of workflow and is unable to run `post` save workflow.

-----------

[PSGO-668]: https://keboola.atlassian.net/browse/PSGO-668?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Matovidlo** [comment]
@michaljurecko @hosekpeter I could not reach the success of `release` step in using `make release`. However the AWS credentials seems to be located elsewhere which should not cause trouble in the workflow anymore.

**Comment by michaljurecko** [comment]
(I think you will manage without me, if you have a specific question, I will answer)

File: .github/workflows/release-cli.yml [hunk 1]
```diff
@@ -444,20 +394,20 @@ jobs:
             --tsaurl http://timestamp.digicert.com \
             --replace \
             $msi
-
+
```
[the remark this answers is not in the document]
Comment (Matovidlo) [reply .github/workflows/release-cli.yml:397]: trailing spaces.

**Comment by Matovidlo** [comment]
The step of `aws-actions/configure-aws-credentials@v4` has side effects that affects other workflows. As it is separate it uses sort of `local` environment in which the configuration of various credentails do not affect others. It should solve this issue https://github.com/keboola/keboola-as-code/actions/runs/9478829184/job/26117435752

File: .github/workflows/release-cli.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Matovidlo) [reply .github/workflows/release-cli.yml:397]: https://github.com/keboola/keboola-as-code/pull/1853 Added draft PR which sets the yaml linter. I would resolve this for now and adjust all yaml code so it is correctly written. (Maintanance task)

**Comment by hosekpeter** [comment]
It's interesting that if it's separated it solves the problem. we have some slack 🧵  where i can learn more about it. @Matovidlo

File: .github/workflows/release-cli.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Matovidlo) [reply .github/workflows/release-cli.yml:463]: You mean add trailing whitespaces? Are you sure? :sweat_smile:

[the remark this answers is not in the document]
Comment (Matovidlo) [reply .github/workflows/release-cli.yml]: I pulled it and see no changes. Weird

**Pull request merged**